client/src/pages/login/volDash/volDash.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './volDash.css';
+
+function volDash() {
```

Comment:
Change to a class, because we will be working with states later on
```suggestion
class VolunteerDashboard {
  render() {
    return <h1>Hello, {this.props.name}</h1>;
  }
}
```

Reply:
Ignore